.github/workflows/python-publish.yml

```diff
@@ -9,23 +9,22 @@ on:
 
```

Comment:
This file does not belong here at all

Reply:
It did at the time because I had to modify it to get the snappy compression to work. I removed that as it was problematic, so this is just the formatting that was done. I'll reset this one.